bucket/nano.json

```diff
@@ -1,32 +1,43 @@
-{
-    "version": "7.1-12",
-    "description": "A small and friendly GNU editor, inspired by Pico",
-    "homepage": "https://www.nano-editor.org/",
-    "license": "GPL-3.0-only",
-    "notes": [
-        "Configure nano by editing its configuration file %USERPROFILE%\\.nanorc",
-        "NOTE: The config file was named nano.rc in the 2.5.3 release, but is now named .nanorc"
-    ],
-    "url": "https://files.lhmouse.com/nano-win/nano-win_10155_v7.1-12-g14f292a7c.7z",
-    "hash": "dce659505c33f6def27048d44d5998dcbe70ae4361c55fe32398af33bc83682e",
-    "post_install": [
-        "if (-not (Test-Path \"$env:USERPROFILE\\.nanorc\")) {",
-        "   Copy-Item \"$dir\\.nanorc\" \"$env:USERPROFILE\\.nanorc\"",
-        "}"
-    ],
-    "checkver": {
-        "url": "https://files.lhmouse.com/nano-win/?C=N&O=D",
-        "regex": "nano-win_(?<build>\\d+)_v([\\d.-]+)-(?<commit>\\w+)\\.7z"
-    },
-    "architecture": {
-        "64bit": {
-            "bin": "pkg_x86_64-w64-mingw32\\bin\\nano.exe"
-        },
-        "32bit": {
-            "bin": "pkg_i686-w64-mingw32\\bin\\nano.exe"
-        }
-    },
-    "autoupdate": {
-        "url": "https://files.lhmouse.com/nano-win/nano-win_$matchBuild_v$version-$matchCommit.7z"
-    }
-}
+{
+    "version": "7.2-8-10163",
+    "description": "The legendary small and friendly GNU editor, now UTF-8 ready for Windows 32/64",
+    "homepage": "https://www.nano-editor.org/",
+    "license": "GPL-3.0-only",
+    "notes": "Configure nano by editing its configuration file with nano ~/.nanorc",
+    "url": "https://github.com/okibcn/nano-for-windows/releases/download/v7.2-8-10163/nano-for-windows_v7.2-8-10163.7z",
+    "hash": "c12b44f9ff0b35cc281cef9926216fd2b360ce4e2c663a5ae14d69414e70600a",
+    "bin": "nano.exe",
+    "architecture": {
+        "64bit": {
+            "pre_install": [
+                "if (-not (Test-Path ~/.nanorc)) {",
+                "    Move-Item \"$dir/.nanorc\" ~/.nanorc",
+                "    Add-Content ~\\.nanorc 'include \"~/scoop/apps/nano/current/syntax/*.nanorc\"'}",
+                "Move-Item \"$dir/pkg_x86_64-w64-mingw32/bin/*\" \"$dir\"",
+                "Move-Item \"$dir/pkg_x86_64-w64-mingw32/share/nano\" \"$dir/syntax\" -Force",
+                "Remove-Item  \"$dir/pkg*\" -Force -Recurse",
+                "(Get-Content ~/.nanorc) -replace '(pkg_.*-w64-mingw32.share.nano)', 'syntax' | Set-Content ~/.nanorc",
+                "(Get-Content ~/.nanorc) -replace '(/usr/share/nano)', '~/scoop/apps/nano/current/syntax' | Set-Content ~/.nanorc"

```

Comment:
These hardcoded `~/` may not work when Scoop is installed elsewhere, customized `ScoopDir` for example.

Reply:
Solved. The code is slighly more complicated as $scoopdir uses backslashes for dir separator and I have to reencode them.

Thanks for the heads up!